packages/config/index.js

```diff
@@ -26,7 +26,7 @@ class ConfigManager extends EventEmitter {
     if (typeof opts.source === 'string') {
       this.fullPath = resolve(opts.source)
     } else {
-      this.fullPath = join(tmpdir(), `platformatic-db-config-${Date.now()}.json`)
+      this.fullPath = join(tmpdir(), `platformatic-config-${Date.now()}.json`)
```

Comment:
```suggestion
      this.fullPath = join(tmpdir(), `platformatic-service-config-${Date.now()}.json`)
```

If it's not a mistake and it should be without `service`, I'm ok.

Reply:
It's not a mistake. config should be generic.